Libraries/dotNetRdf.LinkedDataFragments/dotNetRdf.LinkedDataFragments.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>dotNetRdf.LDF</AssemblyName>
+    <AssemblyTitle>dotNetRdf Linked Data Fragments Support</AssemblyTitle>
+    <RootNamespace>VDS.RDF.LDF</RootNamespace>
```

Comment:
I think I prefer this as the namespace. In the source files you have `VDS.RDF.LinkedPatternFragments` but I think either `VDS.RDF.LDF` or `VDS.RDF.LinkedDataFragments` would be better and in either case should be consistent with this project file declaration.

Reply:
649d4b6